libnmstate/schemas/interfaces.json

```diff
@@ -94,5 +101,41 @@
                 "pattern": "^([a-fA-F0-9]{2}:){5}[a-fA-F0-9]{2}$"
             }
         }
+    },
+    "definitions-ethernet": {
+        "interface": {
+            "allOf": [
+                {
+                    "properties": {
+                        "type": { "type": "string", "enum": ["ethernet"] }
+                    }
+                },
+                {
+                    "oneOf": [
+                        { "$ref": "#/__definitions__/auto-false" },
+                        { "$ref": "#/__definitions__/auto-true" }
+                    ]
+                }
+            ],
+            "required": [ "type", "auto-negotiation" ]
+        }
+    },
+    "__definitions__": {
+        "auto-false": {
```

Comment:
what is this? could you comment on it?

Reply:
So title or description?

Reply:
That is used for the api documentation, not for the schema developer/editor clarity.